src/main/kotlin/com/cognifide/gradle/aem/environment/hosts/HostsOptions.kt

```diff
@@ -0,0 +1,26 @@
+package com.cognifide.gradle.aem.environment.hosts
+
+import org.apache.commons.lang3.SystemUtils
+
+class HostsOptions {
+    var list = listOf(
+            Host("127.0.0.1", "example.com"),
+            Host("127.0.0.1", "demo.example.com"),
+            Host("127.0.0.1", "author.example.com"),
+            Host("127.0.0.1", "invalidation-only")
+    )
+
+    val file = when {
+        SystemUtils.IS_OS_WINDOWS -> HOSTS_FILE_ON_WINDOWS
+        else -> HOSTS_FILE
+    }
+
+    fun configure(config: Map<String, String>) {
+        list = config.map { Host(it.key, it.value) }
+    }
+
+    companion object {
+        const val HOSTS_FILE_ON_WINDOWS = "C:\\Windows\\System32\\drivers\\etc\\hosts"
```

Comment:
this should be a part of `HostsAppenderWindows` /not in shared class

Reply:
True, I was awaken in the middle of the night thinking about this sin ;D Fixed.